components/admin/widgets/form/WidgetsForm.js

```diff
@@ -313,6 +219,12 @@ class WidgetsForm extends React.Component {
     return newForm;
   }
 
+  getWidgetConfig() {
+    return this.onGetWidgetConfig()
+      .then(widgetConfig => widgetConfig)
+      .catch(() => ({}));
```

Comment:
Should we do anything if it fails?

Reply:
Yeah, I didn't have the time to code it yet, it's WIP 😜